cdci_data_analysis/flask_app/schemas.py

```diff
@@ -21,16 +21,17 @@ class Meta:
     query_status = fields.Str(
                         validate=OneOf(["done", "failed", "submitted"]),
                         description="",
-                        required=True
+                        required=False,
+                        missing=""
                     )
 
-    exit_status = fields.Nested(ExitStatus, required=True)
+    exit_status = fields.Nested(ExitStatus, required=False, missing={})
     session_id = fields.Str(required=False) # is it required?
     job_id = fields.Str(required=False) # is it required?
 
     error_message = fields.Str(
-                        validate=OneOf([""]),
                         description="",
-                        required=False # but if present, should be empty
+                        # TODO why should this empty and in which case ?
+                        required=False, # but if present, should be empty
+                        missing=""
```

Comment:
this missing substitute value is never going to be used.
Also you removed the validation which requires no message to be send in this unused field.

Reply:
This was one of the main reasons why the majority of the tests, (those were an error was expected) were not passing. Why should not this field be used?

Reply:
it should not be used because **it is not** used by the clients. Each field in schema is motivated by the client, as explained in the descriptions where they are. This approach should persist and expand.

Reply:
controlling this was the principal purpose of the switch I added in https://github.com/oda-hub/dispatcher-app/pull/199 and this field being used sometimes is probably the main cause of errors not shown correctly in the frontend - assuming that setting  this field has some value for the user at the moment is not useful.